radis/test/io/test_exomol.py

```diff
@@ -28,8 +28,11 @@ def test_exomol_parsing_functions(verbose=True, *args, **kwargs):
     #  Note : this may change if new databases are added ; test would have to
     # be updated in that case.
     databases, recommended = get_exomol_database_list("CH4", "12C-1H4")
-    assert databases == ["xsec-YT10to10", "YT10to10", "YT34to10", "xsec-MM", "MM"]
-    assert recommended == "YT34to10"
+    assert (
+        databases.sort()
```

Comment:
top of my head; i think `databases.sort()` returns None  whereas `sorted(databases)` returns the list